invenio_rdm_records/resources/iiif.py

```diff
@@ -393,3 +397,145 @@ def _rewrite_url(self):
         path = request.path
         path = path.replace(uuid, f"{recid_path}/{filename}.ptif")
         return urljoin(self.server_url, path)
+
+
+class CantaloupeProxy(IIIFProxy):
+    """
+    Proxy for a Cantaloupe instance configured with direct access to RDM's local file bucket.
+
+    Important security note:
+
+    This class is designed to work with a Cantaloupe instance that has full, direct access to RDM's local file bucket
+    To ensure permissions are resoected the Cantaloupe instance *must* only be accessible by RDM.
+    All IIIF requests from end users are proxied through the app, which enforces access control.
+
+    Attributes:
+        iiif_server (str): URL of IIIF server, <scheme>://<server>[:<port>]/ (required).
+        fs_prefix: (str): Prefix for local filestore (required if Cantaloupe's 'FilesystemSource.BasicLookupStrategy.path_prefix'
+            setting doesn't point directly to the root of the instance's local file bucket)
+        chunk_size (int): Chunk size for streaming file content (default: 64KB).
+        fallback (bool): Fall back to RDM IIIF handling on proxy failure (default: False).
+        iiif_version (int): Major IIIF version to use (default: 2).
+        levels (int): Number of levels in the path RDM uses in local filestore, should be loaded from config.
+        timeout (int): Seconds to wait for Cantaloupe to respond (default: 5).
+    """
+
+    def __init__(self):
+        """Initialize the proxy."""
+        self.iiif_server = current_app.config.get("RDM_II'IF_SERVER_URL")
```

Comment:
is the `'` on purpose there?

Reply:
Thanks, seem to have gone over the top when fixing quotes to please black :)